docs/frontend/liquid-tags.md

```diff
@@ -42,3 +42,40 @@ Here is a bunch of liquid tags supported on DEV:
 {% blogcast 1234 %}
 {% kotlin https://pl.kotl.in/owreUFFUG %}
 ```
+## How liquid tags are developed
```

Comment:
should we mention anything about them being iframes sometimes or having to replicate the CSS in other times? The PR linked at the end is an example of a iframe based liquid tag